internal/controller/apiextensions/composite/composition_transforms.go

```diff
@@ -44,16 +44,16 @@ const (
 	errFmtRequiredField                 = "%s is required by type %s"
 	errFmtConvertInputTypeNotSupported  = "invalid input type %T"
 	errFmtConvertFormatPairNotSupported = "conversion from %s to %s is not supported with format %s"
-	errFmtTransformAtIndex              = "transform at index %d returned error"
+	errFmtTransformAtIndex              = "transform %s at index %d returned error"
```

Comment:
```suggestion
	errFmtTransformAtIndex              = "transform type %q at index %d returned error"
```

Rule `#2` in https://github.com/crossplane/crossplane/blob/master/contributing/guide-observability.md#good-errors